filters.py

```diff
@@ -1,2 +1,15 @@
+import re
+
+URL_PATTERN = re.compile(r"(?:http|https):\/\/\S*[^(. )]")
+
+
 def datetimeformat(value, format='%Y-%m-%d'):
     return value.strftime(format)
+
+
+def extendlinks(value):
+    if value:
+        for url_match in URL_PATTERN.finditer(value):
+            url = url_match.group(0)
+            value = value.replace(url, f"<a href='{url}'>{url}</a>")
```

Comment:
if we want to cut the link using CSS, we should add a specific `class` here, shouldn't we? 

```suggestion
            value = value.replace(url, f"<a href='{url}' class='truncated'>{url}</a>")
```
for instance

Reply:
Depends on the outer elements. If we have a unique identifier somewhere telling use we are displaying a specific event, we don't have to.

But I like the idea to make it explicit 👍